src/Nox.Types.EntityFramework/Types/DateTime/DateTimeConverter.cs

```diff
@@ -1,9 +1,13 @@
-using Microsoft.EntityFrameworkCore.Storage.ValueConversion;
+﻿using Microsoft.EntityFrameworkCore.Storage.ValueConversion;
 
 namespace Nox.Types.EntityFramework.Types;
 
-public class DateTimeConverter : ValueConverter<DateTime, System.DateTime>
+public class DateTimeConverter : ValueConverter<DateTimeOffset, System.DateTime>
```

Comment:
Let's do this:
- Remove this DateTimeConverter
- Rename DateTimeRangeConverter into DateTimeOffsetConverter
- Move the DateTimeOffsetConverter from DateTimeRange folder into DateTimeOffset folder
- Use the DateTimeOffsetConverter for the DateTimeDatabaseConfigurator